src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -70,7 +70,7 @@
         "symfony/uid": "^6.4|^7.0",
         "symfony/web-link": "^6.4|^7.0",
         "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0",
-        "twig/twig": "^2.10|^3.0"
+        "twig/twig": "^3.0.4"
```

Comment:
I think we should add a conflict rule, as is already done for symfony components in require-dev.

Reply:
Effectively, the http-kernel package does that for us currently. Shall I duplicate the rule in the bundles' composer.json file?

Reply:
You're right, the constraint is already imposed by http-kernel, which is required by framework-bundle.

There could have been var-dumper which could have this explicit constraint, but the code that calls twig is very robust.

https://github.com/symfony/symfony/blob/d8a4b3c16a08be15aa24e0899bbdcb34c3fef002/src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php#L238